cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -93,6 +95,10 @@ func (m command) build() *cli.Command {
 		},
 	}
 
+	configSource := []altsrc.Sourcer{
+		altsrc.StringSourcer(config.GetConfigFilePath()),
```

Comment:
Note that it's not sufficient to load the config from file since we don't apply the defaults correctly then.